src/code/z_debug.c

```diff
@@ -229,7 +229,7 @@ void func_8006390C(Input* input) {
 
                 if (iREG(0)) {
                     iREG(0) = 0;
-                    func_800AA000(0, iREG(1), iREG(2), iREG(3));
+                    Rumble_Request(0, iREG(1), iREG(2), iREG(3));
```

Comment:
Does this need a 0 to match?, it should be a float

Reply:
I ran into some where it had to be 0 to match, but I was able to fix this one and one in `z_en_horse.c`, however the others seem to be required.